doc/sphinxext/docscrape.py

```diff
@@ -428,7 +429,11 @@ def __init__(self, func, role='func'):
             func, func_name = self.get_func()
             try:
                 # try to read signature
-                argspec = inspect.getargspec(func)
+                if 3 > sys.version_info[0]:
+                    argspec = inspect.getargspec(func)
+                else:
+                    # getargspec deprecated in python3
+                    argspec = inspect.getargspec(func)
```

Comment:
@aabadie No, subsequent usage differs by the '_' in between the two files.  I suppose we could refactor all `argspec` in docscrape.py to `arg_spec` but I'm inclined to leave it as is.